ci/ray_ci/tester.py

```diff
@@ -394,23 +394,47 @@ def _get_test_targets(
     if get_high_impact_tests:
         # run high impact test cases, so we include only high impact tests in the list
         # of targets provided by users
-        high_impact_tests = _get_high_impact_test_targets(team, operating_system)
+        high_impact_tests = _get_high_impact_test_targets(
+            team, operating_system, container
+        )
         final_targets = high_impact_tests.intersection(final_targets)
 
     return list(final_targets)
 
 
-def _get_high_impact_test_targets(team: str, operating_system: str) -> Set[str]:
+def _get_high_impact_test_targets(
+    team: str, operating_system: str, container: TesterContainer
+) -> Set[str]:
     """
     Get all test targets that are high impact
     """
     os_prefix = f"{operating_system}:"
     step_id_to_tests = Test.gen_high_impact_tests(prefix=os_prefix)
-    return {
+    high_impact_tests = {
         test.get_name().lstrip(os_prefix)
         for test in itertools.chain.from_iterable(step_id_to_tests.values())
         if test.get_oncall() == team
     }
+    new_tests = _get_new_tests(os_prefix, container)
+
+    return high_impact_tests.union(new_tests)
+
+
+def _get_new_tests(prefix: str, container: TesterContainer) -> Set[str]:
+    """
+    Get all local test targets that are not in database
+    """
+    local_test_targets = set(
+        container.run_script_with_output(['bazel query "tests(//...)"'])
+        .decode("utf-8")
+        # CUDA image comes with a license header that we need to remove
+        .replace(CUDA_COPYRIGHT, "")
```

Comment:
why do we need to run bazel query in side cuda container?

Reply:
it can run in any container yes; this is the test_in_docker binary though so there is already a container in the life cycle of this script, so i reuse instead of creating a new one